public/assets/css/module.css

```diff
@@ -1,16 +1,15 @@
-
 .code-area {
   background-color: #333333;
   color: #fff;
+  height: 77% !important;
```

Comment:
Adding a selector with a higher specificity will be enough, supposing we want override the bootstrap's class `form-control` , we'd need something like:
```
form-control.code-area {
 ...

}
```
This CSS selector will be targeting an element that will need to have both classes, being more specific than an element having just one of the two classes.